>From 63c201da9da19c832335480393fe79148750a78f Mon Sep 17 00:00:00 2001
From: Salvatore Bonaccorso <carnil@debian.org>
Date: Wed, 31 Jul 2013 20:47:07 +0200
Subject: [PATCH] Document default compressor change for dpkg-deb to xz

Commit 16ab055e95a804a1fc176c7bd72b52a2be8412ab for dpkg 1.17.0 changed
the default compressor from gzip to xz. Adjust manpage for dpkg-deb to
document this change.
---
 man/dpkg-deb.1 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/dpkg-deb.1 b/man/dpkg-deb.1
index f9282e0..5b14370 100644
--- a/man/dpkg-deb.1
+++ b/man/dpkg-deb.1
@@ -230,7 +230,7 @@ gzip (since dpkg 1.17.0) and \fIextreme\fP for xz.
 .BI \-Z compress-type
 Specify which compression type to use when building a package. Allowed
 values are \fIgzip\fP, \fIxz\fP, \fIbzip2\fP, \fIlzma\fP, and \fInone\fP
-(default is \fIgzip\fP).
+(default is \fIxz\fP).
 .TP
 .BI \-\-deb\-format= format
 Set the archive format version used when building (since dpkg 1.17.0).
-- 
1.8.4.rc0

